jsr352/core/src/main/java/org/hibernate/search/jsr352/massindexing/MassIndexingJob.java

```diff
@@ -268,32 +280,32 @@ public ParametersBuilder rowsPerPartition(int rowsPerPartition) {
 		 * Build the parameters.
 		 *
 		 * @return The parameters.
+		 *
```

Comment:
About the cleanup: it's nice (especially the utils stuff), and please leave it in your PR, we will merge it.

But just so you know: in general we avoid this kind of formatting changes (like the empty line above) that have little value, and mess with the commit history without a good reason.
This is not a hard rule, you can change code next to code you edited even if it's not related, but it's the reason why we avoid the use of automatic cleanup in the IDE on already-committed code: it tends to change lots of code in places totally unrelated to the rest of the PR. I don't know if it's what you did... ? I see several changes that I doubt an IDE could do on its own, but I'm mentioning this just in case.